src/gluonts/evaluation/_base.py

```diff
@@ -156,7 +156,7 @@ def __call__(
         ts_iterator = iter(ts_iterator)
         fcst_iterator = iter(fcst_iterator)
 
-        rows = []
+        rows: List = []
```

Comment:
```suggestion
        rows = []
```

Reply:
Let's remove this as well.

Reply:
Sorry, not sure what I did there. Friday nights 😕